documentation/changelog.rst

```diff
@@ -21,6 +21,7 @@ New features
 * Added API endpoints `/sensors/<id>` for fetching a single sensor and `/sensors` (POST) for adding a sensor. [see `PR #759 <https://www.github.com/FlexMeasures/flexmeasures/pull/759>`_] and [see `PR #767 <https://www.github.com/FlexMeasures/flexmeasures/pull/767>`_]
 * The CLI now allows to set lists and dicts as asset & sensor attributes (formerly only single values) [see `PR #762 <https://www.github.com/FlexMeasures/flexmeasures/pull/762>`_]
 * Add `ProcessScheduler` class to optimize the starting time of processes one of the policies developed (INFLEXIBLE, SHIFTABLE and BREAKABLE), accessible via the CLI command `flexmeasures add schedule for-process` [see `PR #729 <https://www.github.com/FlexMeasures/flexmeasures/pull/729>`_ and `PR #768 <https://www.github.com/FlexMeasures/flexmeasures/pull/768>`_]
+* Refactor `Reporter`'s to subclass the `DataGenerator` class which allows storing ``DataSource``'s. [see `PR #751 <https://www.github.com/FlexMeasures/flexmeasures/pull/751>`_] and [see `PR #752 <https://www.github.com/FlexMeasures/flexmeasures/pull/752>`_]
```

Comment:
The entry looks like it belongs to #751 only. Also, the user benefit can be made more explicit. Is it correct to say that with this PR (NB I'm talking about 751 now) *"users will be able to see (e.g. in the UI) exactly which reporter created the report (saved as sensor data), and hosts will be able to identify exactly which configuration was used to create a given report"*?

The changelog entry for this PR (752) should say something about supporting YAML, obviously.

Reply:
Thanks for the suggestion for the changelog, much much better!